srcs/class/route.cpp

```diff
@@ -1,53 +1,173 @@
 #include "route.hpp"
-#include "../webserv.hpp"
 
-Route::Route( std::string location, std::string root ) {
+// default route
+Route::Route(std::string location, std::string root)
+{
 
 	this->root = root;
 	this->location = location;
 	index.push_back("index.html");
 	methods.push_back("GET");
+	this->auto_index = true;
+	this->cgi_enable = false;
```

Comment:
Tu peux utiliser les `auto_index` et `cgi_enabled` en tant qu'options (avec une valeur par défaut, donc facultatifs)! Et en plus tu pourras les ajouter dans les sets de valeurs :D

```suggestion
Route::Route(std::string location, std::string root, bool auto_index = true, bool cgi_enable = false)
{

	this->root = root;
	this->location = location;
	this->auto_index = true;
	this->cgi_enable = false;
```